### PR TITLE
Updates logic for bonus cells

### PR DESCRIPTION
**Pull request by arjun27**
This PR simply updates the logic for bonus cells in the guestbook. It should be completely bug free and worthy of a quick scan + LGTM! 👍Let's get this merged and pushed straight to production 🔥

### CODE REVIEW

File: src/components/Grid/GridLegend.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import React from "react";
 
 import styled from "styled-components";
 
-export function GridLegend() {
+export function GridLegend({ bonusCells = "Bonus" }) {
```
Comment (arjun27) [inline src/components/Grid/GridLegend.js]: Help me double-check that changing this signature won't break anything

File: src/model/signatureMatrix.js [hunk 2]
```diff
@@ -20,8 +20,9 @@ for (let x = 0; x < COLUMNS; x++) {
     const isBonus = isBonusCell(cellNumber);
     const isSpecial = isSpecialCell(cellNumber);
 
-    const signature = signatures.unshift();
-    matrix.push({ isBonus, isSpecial, signature });
+    const signature = signatures.shift();
```
Comment (lostintangent) [inline src/model/signatureMatrix.js]: This change looks good

Comment (lostintangent) [inline src/model/signatureMatrix.js]: This change looks good!

Comment (lostintangent) [inline src/model/signatureMatrix.js]: I just made this same change and pushed straight to mastere. Sorry

Comment (nikmd23) [reply src/model/signatureMatrix.js]: No worries man! Rules are meant to be broken!

Comment (nikmd23) [reply src/model/signatureMatrix.js]: it's alright man, rules are made to be broken.

Comment (nikmd23) [reply src/model/signatureMatrix.js]: Rules are made ot be broken

Comment (nikmd23) [reply src/model/signatureMatrix.js]: YOU DID WAT?

**Pull request closed without merging**